.github/composites/environment_setup/action.yaml

```diff
@@ -71,14 +73,25 @@ runs:
       shell: bash
       run: echo ${{ env.CLOUDSMITH_API_KEY }} | dart pub token add https://dart.cloudsmith.io/alkami/flutter/
 
+    # Install from Github if 'mplat-branch-override' was passed in
     # Updates the git config to allow us to use https github requests (useful when installing mplat directly from github)
-    # - name: Git Config Update
-    #   shell: bash
-    #   run: git config --global url."https://x-token-auth:${{ env.GITHUB_TOKEN }}@github.com".InsteadOf https://github.com
+    - name: Git Config Update
+      if: ${{ inputs.mplat-branch-override != '' }}
+      shell: bash
+      run: git config --global url."https://x-token-auth:${{ env.GITHUB_TOKEN }}@github.com".InsteadOf https://github.com
     
-    - name: Activate mplat
+    # Install from Github if 'mplat-branch-override' was passed in
+    - name: Activate mplat from Github branch 
+      if: ${{ inputs.mplat-branch-override != '' }}
+      shell: bash
+      run: |
+        dart pub global activate --source git https://github.com/Alkami/mobile_platform_cli.git \
+          --git-ref ${{ inputs.mplat-branch-override }}
```

Comment:
I didn't even consider `--git-ref` supported anything other than a hash but it makes sense!